src/uikit.cs

```diff
@@ -7986,35 +8207,81 @@ interface UIButton : UIAccessibilityContentSizeCategoryImageAdjusting
 		[Export ("buttonWithType:primaryAction:")]
 		UIButton FromType (UIButtonType buttonType, [NullAllowed] UIAction primaryAction);
 
+		[TV (15,0), NoWatch, iOS (15,0), MacCatalyst (15,0)]
+		[Static]
+		[Export ("buttonWithConfiguration:primaryAction:")]
+		UIButton GetButton (UIButtonConfiguration configuration, [NullAllowed] UIAction primaryAction);
+
+		[TV (15,0), NoWatch, iOS (15,0), MacCatalyst (15,0)]
+		[NullAllowed, Export ("configuration", ArgumentSemantic.Copy)]
+		UIButtonConfiguration Configuration { get; set; }
+
+		[TV (15,0), NoWatch, iOS (15,0), MacCatalyst (15,0)]
+		[Export ("setNeedsUpdateConfiguration")]
+		void SetNeedsUpdateConfiguration ();
+
+		[TV (15,0), NoWatch, iOS (15,0), MacCatalyst (15,0)]
+		[Export ("updateConfiguration")]
+		void UpdateConfiguration ();
+
+		[TV (15,0), NoWatch, iOS (15,0), MacCatalyst (15,0)]
+		[NullAllowed, Export ("configurationUpdateHandler", ArgumentSemantic.Copy)]
+		UIButtonConfigurationUpdateHandler ConfigurationUpdateHandler { get; set; }
+
+		[TV (15,0), NoWatch, iOS (15,0), MacCatalyst (15,0)]
+		[Export ("automaticallyUpdatesConfiguration")]
+		bool AutomaticallyUpdatesConfiguration { get; set; }
+
 		[Export ("buttonWithType:")] [Static]
 		UIButton FromType (UIButtonType type);
 
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Ignored when 'UIButtonConfiguration' is used.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Ignored when 'UIButtonConfiguration' is used.")]
 		[Appearance]
 		[Export ("contentEdgeInsets")]
 		UIEdgeInsets ContentEdgeInsets {get;set;}
 
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Ignored when 'UIButtonConfiguration' is used.")]
+		[Deprecated (PlatformName.TvOS, 15, 0, message: "Ignored when 'UIButtonConfiguration' is used.")]
```

Comment:
It's not deprecated on the watch?

Reply:
This type is not available on Watch

```
#if !WATCH
	[BaseType (typeof (UIControl))]
	interface UIButton : UIAccessibilityContentSizeCategoryImageAdjusting
#if IOS
```

Reply:
Then the [NoWatch] attribute on the other members added to this type is redundant.

Reply:
some members, a few lines down, are decorated `[Watch (8,0)]`